### PR TITLE
Improved helper library loading error message

### PR DESCRIPTION
**Pull request by ardraeiss**
### PURPOSE
It's unclear from Blender error message what addon version has issue with helper library loading. This PR expands the OSError traceback with addon, core and RIF version numbers and library path so it would be easier to figure out loading issue reasong from exception window screenshot.

### EFFECT OF CHANGE
- added addon, RPR and RIF version numbers and the addon helper library path to the library loading exception.

### TECHNICAL STEPS
- raised Exception on Helper lib OSError loading exception with versions info;
- attached it to the original exception traceback.

### NOTES
The result will look like this
image
Maybe it's too big.

### CODE REVIEW

File: src/rprblender/utils/helper_lib.py [hunk 1]
```diff
@@ -60,7 +61,15 @@ def init():
 
     lib_path = next(p / lib_name for p in paths if (p / lib_name).is_file())
     log('Load lib', lib_path)
-    lib = ctypes.cdll.LoadLibrary(str(lib_path))
+    try:
+        lib = ctypes.cdll.LoadLibrary(str(lib_path))
+    except OSError as e:  # expand the traceback info with the exact addon version and library name
+        raise Exception(f"Failed to load library {lib_path}",
```
Comment (bnagirniak) [inline src/rprblender/utils/helper_lib.py:67]: What do you think of just logging this error and provide plugin to be in working state? Of course openvdb and sun&sky functionality won't be working, but everything else will be working. 
@bsavery what do you think about it?

**Review by bnagirniak — COMMENTED** [review]
Added improvement comment question.

File: src/rprblender/utils/helper_lib.py (hunk 1 above)
Comment (bsavery) [reply src/rprblender/utils/helper_lib.py:67]: I mean, it depends on the library.  If RPR lib fails that's bad.  But hard to tell if other libraries don't fail what is going to work and what won't

Comment (bsavery) [reply src/rprblender/utils/helper_lib.py:67]: So a "borked" load is hard to predict.  and might be ok just to fail

**Review by bnagirniak — APPROVED** [review]

**Pull request merged**